prestoadmin/util/application.py

```diff
@@ -14,26 +14,27 @@
 
 """Logic at the application level for logging and exception handling"""
 
-from prestoadmin import __version__
-from prestoadmin.util import constants
-from prestoadmin.util import filesystem
-from prestoadmin.util.exception import ExceptionWithCause
-
-import __main__ as main
-
 import functools
 import logging
 import logging.config
 import os
 import sys
 import traceback
 
+import __main__ as main
+
+from prestoadmin import __version__
+from prestoadmin.util import constants
+from prestoadmin.util import filesystem
+from prestoadmin.util.exception import ExceptionWithCause
 
 # Normally this would use the logger for __name__, however, this is
 # effectively the "root" logger for the application.  If this code
 # were running directly in the executable script __name__ would be
 # set to '__main__', so we emulate that same behavior here.  This should
 # resolve to the same logger that will be used by the entry point script.
+from prestoadmin.util.local_config_util import get_log_directory
```

Comment:
You probably don't want this here as the comment should be directly above the line it is qualifying.